service/history/workflow/state_transition_history.go

```diff
@@ -143,3 +143,28 @@ func CompareVersionedTransition(
 
 	return 0
 }
+
+func CopyVersionedTransition(
```

Comment:
I wonder if common.CloneProto will do the same thing.

Reply:
updated with common.CloneProto